addons/website_blog/models/website_blog.py

```diff
@@ -257,6 +257,11 @@ def _notify_record_by_inbox(self, message, recipients_data, msg_vals=False, **kw
     def _default_website_meta(self):
         res = super(BlogPost, self)._default_website_meta()
         res['default_opengraph']['og:description'] = res['default_twitter']['twitter:description'] = self.subtitle
+        res['default_opengraph']['og:type'] = 'article'
+        res['default_opengraph']['article:published_time'] = self.post_date
+        res['default_opengraph']['article:modified_time'] = self.write_date
+        res['default_opengraph']['article:author'] = self.author_id.sudo().name
```

Comment:
why this sudo ?

Reply:
Runbot failed before as there is no access to res.partner allowed

Reply:
@JKE-be let me know if you need anything else.

Reply:
still some doubt about the author tag.

I dont find where it is display with twitter / facebook /...
And the specs is ambiguous

btw it should be an url which contain profile og.

but facebook wait a facabook profile id :/

Reply:
It's not visually shown on Twitter etc but it is added in the tags and does influence SEO. FB does want a Facebook profile ID but there's nothing we can do to get that in like FB wants it though.

Reply:
So what do you think @JKE-be?

Reply:
tha add something wrong without known the real impact is strange.

I will remove it for now

and maybe come back with og:probilfe on user/pofile page

Reply:
so want me to remove the `author` tag and force update so you can merge then?

Reply:
@JKE-be friendly reminder, let me know if you need anything else from me!

Reply:
yes, plz remove author since the spec form ogp.me say " article:author - profile array - Writers of the article." what we don't have at this moment.
thx